src/xcode/ENA/ENA/Resources/Storyboards/InviteFriends.storyboard

```diff
@@ -16,14 +16,14 @@
                         <rect key="frame" x="0.0" y="0.0" width="414" height="896"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
-                            <scrollView clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="scaleToFill" showsHorizontalScrollIndicator="NO" translatesAutoresizingMaskIntoConstraints="NO" id="DvA-ys-EYb">
+                            <scrollView clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="scaleToFill" alwaysBounceVertical="YES" showsHorizontalScrollIndicator="NO" translatesAutoresizingMaskIntoConstraints="NO" id="DvA-ys-EYb">
```

Comment:
@Bohne9 Can you please limit your PR to these lines?
We don't have to commit local frame changes. Only causes merge conflicts, if not required.

Reply:
@EisvogelSAP Are you sure that the other lines are not required? I only changed one checkmark in the storyboard file and Xcode made these changes.

Reply:
Yes, unfortunately Xcode does this. Some of those frame should be updates, as the are wrong. But it is not required for the change to take effect. Will update storyboard frame at some point to align design time with compile time frames. But for now we have to keep merge conflicts at a minimum, please.

Reply:
I think you will have to help me real quick. How can I limit the changes of the PR to the two lines? Never done that before.

Reply:
Can you give me write access to your repo? Then I can do the required changes. Otherwise just google for "git commit single line" or similar. Best to use a GUI if you are not familiar with the CLI.

Reply:
I've sent you an invitation.

Reply:
Okay never mind I think I got it.